lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1129,6 +1129,23 @@ def test_get_html_hls_no_video_id(self):
         context = self.item_descriptor.render(STUDENT_VIEW).content
         assert "'download_video_link': None" in context
 
+
```

Comment:
@sathiscode A double empty line here will probably cause the linter check to fail during the CI. Can you kindly check and remove the extra line if present?